[2022-01-29 13:43:59,110] {processor.py:163} INFO - Started process (PID=1529) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:43:59,111] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:43:59,112] {logging_mixin.py:109} INFO - [2022-01-29 13:43:59,112] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:44:00,000] {logging_mixin.py:109} INFO - [2022-01-29 13:43:59,998] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114, in <module>
    "sourceUris": [f"gs://{BUCKET}/raw/{parquet_file}"],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 177, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['dataset_id'] is required
[2022-01-29 13:44:00,016] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:44:00,025] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.920 seconds
[2022-01-29 13:44:30,356] {processor.py:163} INFO - Started process (PID=1560) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:44:30,360] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:44:30,361] {logging_mixin.py:109} INFO - [2022-01-29 13:44:30,361] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:44:31,273] {logging_mixin.py:109} INFO - [2022-01-29 13:44:31,272] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114, in <module>
    "sourceUris": [f"gs://{BUCKET}/raw/{parquet_file}"],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 177, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['dataset_id'] is required
[2022-01-29 13:44:31,290] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:44:31,299] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.948 seconds
[2022-01-29 13:45:01,464] {processor.py:163} INFO - Started process (PID=1600) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:45:01,465] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:45:01,466] {logging_mixin.py:109} INFO - [2022-01-29 13:45:01,466] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:45:02,313] {logging_mixin.py:109} INFO - [2022-01-29 13:45:02,311] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114, in <module>
    "sourceUris": [f"gs://{BUCKET}/raw/{parquet_file}"],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 177, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['dataset_id'] is required
[2022-01-29 13:45:02,327] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:45:02,335] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.876 seconds
[2022-01-29 13:45:20,700] {processor.py:163} INFO - Started process (PID=1620) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:45:20,702] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:45:20,703] {logging_mixin.py:109} INFO - [2022-01-29 13:45:20,703] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:45:21,618] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:45:21,826] {logging_mixin.py:109} INFO - [2022-01-29 13:45:21,825] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:45:21,843] {logging_mixin.py:109} INFO - [2022-01-29 13:45:21,843] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag to 2022-01-02T06:00:00+00:00
[2022-01-29 13:45:21,863] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.168 seconds
[2022-01-29 13:45:52,025] {processor.py:163} INFO - Started process (PID=1661) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:45:52,027] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:45:52,028] {logging_mixin.py:109} INFO - [2022-01-29 13:45:52,028] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:45:52,877] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:45:52,887] {logging_mixin.py:109} INFO - [2022-01-29 13:45:52,886] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:45:52,905] {logging_mixin.py:109} INFO - [2022-01-29 13:45:52,905] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag to 2022-01-02T06:00:00+00:00
[2022-01-29 13:45:52,916] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.895 seconds
[2022-01-29 13:46:23,284] {processor.py:163} INFO - Started process (PID=1692) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:46:23,287] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:46:23,288] {logging_mixin.py:109} INFO - [2022-01-29 13:46:23,288] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:46:24,308] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:46:24,312] {logging_mixin.py:109} INFO - [2022-01-29 13:46:24,311] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:46:24,329] {logging_mixin.py:109} INFO - [2022-01-29 13:46:24,328] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag to 2022-01-02T06:00:00+00:00
[2022-01-29 13:46:24,339] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.062 seconds
[2022-01-29 13:46:48,596] {processor.py:163} INFO - Started process (PID=1715) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:46:48,600] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:46:48,602] {logging_mixin.py:109} INFO - [2022-01-29 13:46:48,602] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:46:49,898] {processor.py:690} INFO - Deactivated 1 DAGs which are no longer present in /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:46:49,898] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:46:50,183] {logging_mixin.py:109} INFO - [2022-01-29 13:46:50,182] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:46:50,191] {logging_mixin.py:109} INFO - [2022-01-29 13:46:50,190] {dag.py:2415} INFO - Creating ORM DAG for data_ingestion_gcs_dag_2
[2022-01-29 13:46:50,202] {logging_mixin.py:109} INFO - [2022-01-29 13:46:50,202] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2021-01-02 06:00:00+00:00
[2022-01-29 13:46:50,223] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.635 seconds
[2022-01-29 13:47:20,709] {processor.py:163} INFO - Started process (PID=1758) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:47:20,711] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:47:20,712] {logging_mixin.py:109} INFO - [2022-01-29 13:47:20,712] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:47:21,587] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:47:21,597] {logging_mixin.py:109} INFO - [2022-01-29 13:47:21,596] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:47:21,613] {logging_mixin.py:109} INFO - [2022-01-29 13:47:21,613] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2021-01-02 06:00:00+00:00
[2022-01-29 13:47:21,624] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.919 seconds
[2022-01-29 13:47:30,644] {processor.py:163} INFO - Started process (PID=1778) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:47:30,646] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:47:30,647] {logging_mixin.py:109} INFO - [2022-01-29 13:47:30,647] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:47:31,633] {logging_mixin.py:109} INFO - [2022-01-29 13:47:31,631] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114, in <module>
    "sourceUris": [f"gs://{BUCKET}/raw/{parquet_file}"],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 177, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['dataset_id'] is required
[2022-01-29 13:47:31,650] {processor.py:690} INFO - Deactivated 1 DAGs which are no longer present in /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:47:31,650] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:47:31,662] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.024 seconds
[2022-01-29 13:48:01,749] {processor.py:163} INFO - Started process (PID=1808) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:48:01,752] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:48:01,753] {logging_mixin.py:109} INFO - [2022-01-29 13:48:01,753] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:48:02,697] {logging_mixin.py:109} INFO - [2022-01-29 13:48:02,695] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114, in <module>
    "sourceUris": [f"gs://{BUCKET}/raw/{parquet_file}"],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 177, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['dataset_id'] is required
[2022-01-29 13:48:02,717] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:48:02,733] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.989 seconds
[2022-01-29 13:48:32,873] {processor.py:163} INFO - Started process (PID=1849) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:48:32,874] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:48:32,875] {logging_mixin.py:109} INFO - [2022-01-29 13:48:32,875] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:48:33,815] {logging_mixin.py:109} INFO - [2022-01-29 13:48:33,814] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114, in <module>
    "sourceUris": [f"gs://{BUCKET}/raw/{parquet_file}"],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 177, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['dataset_id'] is required
[2022-01-29 13:48:33,832] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:48:33,844] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.976 seconds
[2022-01-29 13:48:58,091] {processor.py:163} INFO - Started process (PID=1881) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:48:58,093] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:48:58,094] {logging_mixin.py:109} INFO - [2022-01-29 13:48:58,094] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:48:58,904] {logging_mixin.py:109} INFO - [2022-01-29 13:48:58,903] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 118, in <module>
    "sourceUris": [f"gs://{BUCKET}/raw/{parquet_file}"],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 177, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['dataset_id'] is required
[2022-01-29 13:48:58,920] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:48:58,928] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.842 seconds
[2022-01-29 13:49:01,363] {processor.py:163} INFO - Started process (PID=1886) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:01,364] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:49:01,365] {logging_mixin.py:109} INFO - [2022-01-29 13:49:01,365] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:01,369] {logging_mixin.py:109} INFO - [2022-01-29 13:49:01,368] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114
    task_id="bigquery_external_table_task",
    ^
SyntaxError: keyword argument repeated
[2022-01-29 13:49:01,390] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:01,400] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.042 seconds
[2022-01-29 13:49:03,372] {processor.py:163} INFO - Started process (PID=1887) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:03,374] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:49:03,374] {logging_mixin.py:109} INFO - [2022-01-29 13:49:03,374] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:03,378] {logging_mixin.py:109} INFO - [2022-01-29 13:49:03,377] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114
    task_id="bigquery_external_table_task",
    ^
SyntaxError: keyword argument repeated
[2022-01-29 13:49:03,402] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:03,415] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.047 seconds
[2022-01-29 13:49:06,436] {processor.py:163} INFO - Started process (PID=1888) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:06,438] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:49:06,438] {logging_mixin.py:109} INFO - [2022-01-29 13:49:06,438] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:06,442] {logging_mixin.py:109} INFO - [2022-01-29 13:49:06,441] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114
    task_id="bigquery_external_table_task",
    ^
SyntaxError: keyword argument repeated
[2022-01-29 13:49:06,466] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:06,477] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.045 seconds
[2022-01-29 13:49:15,516] {processor.py:163} INFO - Started process (PID=1899) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:15,518] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:49:15,518] {logging_mixin.py:109} INFO - [2022-01-29 13:49:15,518] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:15,522] {logging_mixin.py:109} INFO - [2022-01-29 13:49:15,521] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114
    task_id="bigquery_external_table_task",
    ^
SyntaxError: keyword argument repeated
[2022-01-29 13:49:15,545] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:15,556] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.045 seconds
[2022-01-29 13:49:45,664] {processor.py:163} INFO - Started process (PID=1935) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:45,668] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:49:45,669] {logging_mixin.py:109} INFO - [2022-01-29 13:49:45,669] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:45,674] {logging_mixin.py:109} INFO - [2022-01-29 13:49:45,672] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 114
    task_id="bigquery_external_table_task",
    ^
SyntaxError: keyword argument repeated
[2022-01-29 13:49:45,694] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:45,708] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.048 seconds
[2022-01-29 13:49:50,691] {processor.py:163} INFO - Started process (PID=1938) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:50,692] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:49:50,693] {logging_mixin.py:109} INFO - [2022-01-29 13:49:50,693] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:50,696] {logging_mixin.py:109} INFO - [2022-01-29 13:49:50,695] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 115
    task_id="bigquery_external_table_task",
    ^
SyntaxError: keyword argument repeated
[2022-01-29 13:49:50,716] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:49:50,726] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.040 seconds
[2022-01-29 13:50:05,726] {processor.py:163} INFO - Started process (PID=1954) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:05,728] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:50:05,729] {logging_mixin.py:109} INFO - [2022-01-29 13:50:05,728] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:05,732] {logging_mixin.py:109} INFO - [2022-01-29 13:50:05,731] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 115
    task_id="bigquery_external_table_task",
    ^
SyntaxError: keyword argument repeated
[2022-01-29 13:50:05,751] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:05,884] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.162 seconds
[2022-01-29 13:50:08,787] {processor.py:163} INFO - Started process (PID=1955) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:08,788] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:50:08,789] {logging_mixin.py:109} INFO - [2022-01-29 13:50:08,789] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:08,793] {logging_mixin.py:109} INFO - [2022-01-29 13:50:08,792] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 115
    task_id="bigquery_external_table_task",
    ^
SyntaxError: keyword argument repeated
[2022-01-29 13:50:08,816] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:08,969] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.188 seconds
[2022-01-29 13:50:14,001] {processor.py:163} INFO - Started process (PID=1968) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:14,003] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:50:14,004] {logging_mixin.py:109} INFO - [2022-01-29 13:50:14,004] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:14,008] {logging_mixin.py:109} INFO - [2022-01-29 13:50:14,007] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 115
    task_id="bigquery_external_table_task",
    ^
SyntaxError: keyword argument repeated
[2022-01-29 13:50:14,031] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:14,177] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.182 seconds
[2022-01-29 13:50:38,227] {processor.py:163} INFO - Started process (PID=1993) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:38,232] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:50:38,233] {logging_mixin.py:109} INFO - [2022-01-29 13:50:38,233] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:38,238] {logging_mixin.py:109} INFO - [2022-01-29 13:50:38,237] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_gcs_dag_2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_gcs_dag_2.py", line 115
    task_id="bigquery_external_table_task",
    ^
SyntaxError: keyword argument repeated
[2022-01-29 13:50:38,386] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:38,395] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.172 seconds
[2022-01-29 13:50:42,282] {processor.py:163} INFO - Started process (PID=1994) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:42,283] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:50:42,284] {logging_mixin.py:109} INFO - [2022-01-29 13:50:42,284] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:43,153] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:50:43,360] {logging_mixin.py:109} INFO - [2022-01-29 13:50:43,360] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:50:43,376] {logging_mixin.py:109} INFO - [2022-01-29 13:50:43,376] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2021-01-02 06:00:00+00:00
[2022-01-29 13:50:43,395] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.118 seconds
[2022-01-29 13:51:14,379] {processor.py:163} INFO - Started process (PID=2033) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:51:15,626] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:51:16,563] {logging_mixin.py:109} INFO - [2022-01-29 13:51:16,563] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:51:17,421] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:51:17,432] {logging_mixin.py:109} INFO - [2022-01-29 13:51:17,431] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:51:17,453] {logging_mixin.py:109} INFO - [2022-01-29 13:51:17,453] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:51:20,816] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 6.441 seconds
[2022-01-29 13:51:50,918] {processor.py:163} INFO - Started process (PID=2090) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:51:50,926] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:51:50,927] {logging_mixin.py:109} INFO - [2022-01-29 13:51:50,927] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:51:53,323] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:51:53,353] {logging_mixin.py:109} INFO - [2022-01-29 13:51:53,352] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:51:53,401] {logging_mixin.py:109} INFO - [2022-01-29 13:51:53,400] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:51:53,435] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 2.537 seconds
[2022-01-29 13:52:23,489] {processor.py:163} INFO - Started process (PID=2129) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:52:23,493] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:52:23,495] {logging_mixin.py:109} INFO - [2022-01-29 13:52:23,494] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:52:24,443] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:52:24,458] {logging_mixin.py:109} INFO - [2022-01-29 13:52:24,458] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:52:24,490] {logging_mixin.py:109} INFO - [2022-01-29 13:52:24,490] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:52:24,505] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.022 seconds
[2022-01-29 13:52:54,606] {processor.py:163} INFO - Started process (PID=2165) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:52:54,608] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:52:54,609] {logging_mixin.py:109} INFO - [2022-01-29 13:52:54,608] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:52:55,573] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:52:55,585] {logging_mixin.py:109} INFO - [2022-01-29 13:52:55,585] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:52:55,604] {logging_mixin.py:109} INFO - [2022-01-29 13:52:55,604] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:52:55,618] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.018 seconds
[2022-01-29 13:53:25,808] {processor.py:163} INFO - Started process (PID=2208) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:53:25,810] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:53:25,811] {logging_mixin.py:109} INFO - [2022-01-29 13:53:25,811] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:53:26,699] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:53:26,715] {logging_mixin.py:109} INFO - [2022-01-29 13:53:26,714] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:53:26,733] {logging_mixin.py:109} INFO - [2022-01-29 13:53:26,733] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:53:26,746] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.944 seconds
[2022-01-29 13:53:35,812] {processor.py:163} INFO - Started process (PID=2225) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:53:35,813] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:53:35,814] {logging_mixin.py:109} INFO - [2022-01-29 13:53:35,813] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:53:36,628] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:53:36,829] {logging_mixin.py:109} INFO - [2022-01-29 13:53:36,828] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:53:36,846] {logging_mixin.py:109} INFO - [2022-01-29 13:53:36,846] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:53:36,860] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.053 seconds
[2022-01-29 13:54:07,029] {processor.py:163} INFO - Started process (PID=2257) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:54:07,032] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:54:07,033] {logging_mixin.py:109} INFO - [2022-01-29 13:54:07,033] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:54:08,318] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:54:08,333] {logging_mixin.py:109} INFO - [2022-01-29 13:54:08,333] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:54:08,360] {logging_mixin.py:109} INFO - [2022-01-29 13:54:08,360] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:54:08,379] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.356 seconds
[2022-01-29 13:54:39,156] {processor.py:163} INFO - Started process (PID=2295) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:54:39,158] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:54:39,159] {logging_mixin.py:109} INFO - [2022-01-29 13:54:39,159] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:54:39,947] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:54:39,957] {logging_mixin.py:109} INFO - [2022-01-29 13:54:39,956] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:54:39,982] {logging_mixin.py:109} INFO - [2022-01-29 13:54:39,981] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:54:39,996] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.844 seconds
[2022-01-29 13:55:10,457] {processor.py:163} INFO - Started process (PID=2324) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:55:10,462] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:55:10,463] {logging_mixin.py:109} INFO - [2022-01-29 13:55:10,463] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:55:11,313] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:55:11,324] {logging_mixin.py:109} INFO - [2022-01-29 13:55:11,324] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:55:11,344] {logging_mixin.py:109} INFO - [2022-01-29 13:55:11,343] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:55:11,357] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.906 seconds
[2022-01-29 13:55:41,615] {processor.py:163} INFO - Started process (PID=2364) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:55:41,619] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:55:41,620] {logging_mixin.py:109} INFO - [2022-01-29 13:55:41,620] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:55:42,507] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:55:42,518] {logging_mixin.py:109} INFO - [2022-01-29 13:55:42,518] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:55:42,537] {logging_mixin.py:109} INFO - [2022-01-29 13:55:42,537] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:55:42,549] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.939 seconds
[2022-01-29 13:56:12,764] {processor.py:163} INFO - Started process (PID=2395) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:56:12,768] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:56:12,769] {logging_mixin.py:109} INFO - [2022-01-29 13:56:12,769] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:56:13,621] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:56:13,632] {logging_mixin.py:109} INFO - [2022-01-29 13:56:13,631] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:56:13,649] {logging_mixin.py:109} INFO - [2022-01-29 13:56:13,649] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:56:13,660] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.901 seconds
[2022-01-29 13:56:44,011] {processor.py:163} INFO - Started process (PID=2439) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:56:44,015] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:56:44,016] {logging_mixin.py:109} INFO - [2022-01-29 13:56:44,016] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:56:44,798] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:56:44,808] {logging_mixin.py:109} INFO - [2022-01-29 13:56:44,807] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:56:44,824] {logging_mixin.py:109} INFO - [2022-01-29 13:56:44,824] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:56:44,834] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.828 seconds
[2022-01-29 13:57:15,114] {processor.py:163} INFO - Started process (PID=2468) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:57:15,117] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:57:15,118] {logging_mixin.py:109} INFO - [2022-01-29 13:57:15,117] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:57:16,032] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:57:16,043] {logging_mixin.py:109} INFO - [2022-01-29 13:57:16,043] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:57:16,064] {logging_mixin.py:109} INFO - [2022-01-29 13:57:16,064] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:57:16,080] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.971 seconds
[2022-01-29 13:57:46,268] {processor.py:163} INFO - Started process (PID=2507) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:57:46,270] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:57:46,271] {logging_mixin.py:109} INFO - [2022-01-29 13:57:46,271] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:57:47,041] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:57:47,051] {logging_mixin.py:109} INFO - [2022-01-29 13:57:47,050] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:57:47,067] {logging_mixin.py:109} INFO - [2022-01-29 13:57:47,067] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:57:47,077] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.814 seconds
[2022-01-29 13:58:17,629] {processor.py:163} INFO - Started process (PID=2539) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:58:17,633] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:58:17,633] {logging_mixin.py:109} INFO - [2022-01-29 13:58:17,633] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:58:18,442] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:58:18,452] {logging_mixin.py:109} INFO - [2022-01-29 13:58:18,451] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:58:18,469] {logging_mixin.py:109} INFO - [2022-01-29 13:58:18,469] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:58:18,480] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.856 seconds
[2022-01-29 13:58:48,650] {processor.py:163} INFO - Started process (PID=2581) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:58:48,652] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:58:48,654] {logging_mixin.py:109} INFO - [2022-01-29 13:58:48,653] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:58:49,494] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:58:49,503] {logging_mixin.py:109} INFO - [2022-01-29 13:58:49,503] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:58:49,523] {logging_mixin.py:109} INFO - [2022-01-29 13:58:49,523] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:58:49,541] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.897 seconds
[2022-01-29 13:59:19,876] {processor.py:163} INFO - Started process (PID=2614) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:59:19,879] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:59:19,880] {logging_mixin.py:109} INFO - [2022-01-29 13:59:19,880] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:59:20,708] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:59:20,719] {logging_mixin.py:109} INFO - [2022-01-29 13:59:20,718] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:59:20,737] {logging_mixin.py:109} INFO - [2022-01-29 13:59:20,737] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:59:20,748] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.876 seconds
[2022-01-29 13:59:51,090] {processor.py:163} INFO - Started process (PID=2654) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:59:51,091] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 13:59:51,092] {logging_mixin.py:109} INFO - [2022-01-29 13:59:51,092] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:59:52,034] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 13:59:52,049] {logging_mixin.py:109} INFO - [2022-01-29 13:59:52,048] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 13:59:52,076] {logging_mixin.py:109} INFO - [2022-01-29 13:59:52,075] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 13:59:52,090] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.005 seconds
[2022-01-29 14:00:22,225] {processor.py:163} INFO - Started process (PID=2685) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:00:22,227] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:00:22,228] {logging_mixin.py:109} INFO - [2022-01-29 14:00:22,228] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:00:23,064] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:00:23,074] {logging_mixin.py:109} INFO - [2022-01-29 14:00:23,074] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:00:23,091] {logging_mixin.py:109} INFO - [2022-01-29 14:00:23,091] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:00:23,103] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.883 seconds
[2022-01-29 14:00:53,289] {processor.py:163} INFO - Started process (PID=2726) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:00:53,290] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:00:53,291] {logging_mixin.py:109} INFO - [2022-01-29 14:00:53,291] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:00:54,115] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:00:54,129] {logging_mixin.py:109} INFO - [2022-01-29 14:00:54,129] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:00:54,156] {logging_mixin.py:109} INFO - [2022-01-29 14:00:54,156] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:00:54,170] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.886 seconds
[2022-01-29 14:01:24,430] {processor.py:163} INFO - Started process (PID=2758) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:01:24,433] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:01:24,434] {logging_mixin.py:109} INFO - [2022-01-29 14:01:24,434] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:01:25,353] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:01:25,367] {logging_mixin.py:109} INFO - [2022-01-29 14:01:25,367] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:01:25,394] {logging_mixin.py:109} INFO - [2022-01-29 14:01:25,393] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:01:25,411] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.986 seconds
[2022-01-29 14:01:55,673] {processor.py:163} INFO - Started process (PID=2798) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:01:55,679] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:01:55,684] {logging_mixin.py:109} INFO - [2022-01-29 14:01:55,683] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:01:58,837] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:01:58,896] {logging_mixin.py:109} INFO - [2022-01-29 14:01:58,893] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:01:58,973] {logging_mixin.py:109} INFO - [2022-01-29 14:01:58,972] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:01:59,018] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 3.371 seconds
[2022-01-29 14:02:29,875] {processor.py:163} INFO - Started process (PID=2830) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:02:29,878] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:02:29,879] {logging_mixin.py:109} INFO - [2022-01-29 14:02:29,878] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:02:30,713] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:02:30,726] {logging_mixin.py:109} INFO - [2022-01-29 14:02:30,725] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:02:30,745] {logging_mixin.py:109} INFO - [2022-01-29 14:02:30,744] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:02:30,758] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.887 seconds
[2022-01-29 14:03:01,068] {processor.py:163} INFO - Started process (PID=2861) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:03:01,070] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:03:01,071] {logging_mixin.py:109} INFO - [2022-01-29 14:03:01,070] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:03:01,878] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:03:01,887] {logging_mixin.py:109} INFO - [2022-01-29 14:03:01,887] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:03:01,904] {logging_mixin.py:109} INFO - [2022-01-29 14:03:01,903] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:03:01,914] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.851 seconds
[2022-01-29 14:03:32,268] {processor.py:163} INFO - Started process (PID=2904) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:03:32,271] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:03:32,272] {logging_mixin.py:109} INFO - [2022-01-29 14:03:32,272] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:03:33,156] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:03:33,167] {logging_mixin.py:109} INFO - [2022-01-29 14:03:33,167] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:03:33,185] {logging_mixin.py:109} INFO - [2022-01-29 14:03:33,185] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:03:33,196] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.932 seconds
[2022-01-29 14:04:03,521] {processor.py:163} INFO - Started process (PID=2937) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:04:03,523] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:04:03,523] {logging_mixin.py:109} INFO - [2022-01-29 14:04:03,523] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:04:04,381] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:04:04,391] {logging_mixin.py:109} INFO - [2022-01-29 14:04:04,391] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:04:04,408] {logging_mixin.py:109} INFO - [2022-01-29 14:04:04,408] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:04:04,419] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.903 seconds
[2022-01-29 14:04:34,678] {processor.py:163} INFO - Started process (PID=2977) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:04:34,680] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:04:34,681] {logging_mixin.py:109} INFO - [2022-01-29 14:04:34,681] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:04:35,536] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:04:35,546] {logging_mixin.py:109} INFO - [2022-01-29 14:04:35,546] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:04:35,564] {logging_mixin.py:109} INFO - [2022-01-29 14:04:35,563] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:04:35,574] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.900 seconds
[2022-01-29 14:05:05,859] {processor.py:163} INFO - Started process (PID=3007) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:05:05,861] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:05:05,862] {logging_mixin.py:109} INFO - [2022-01-29 14:05:05,862] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:05:06,657] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:05:06,668] {logging_mixin.py:109} INFO - [2022-01-29 14:05:06,667] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:05:06,686] {logging_mixin.py:109} INFO - [2022-01-29 14:05:06,686] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:05:06,698] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.844 seconds
[2022-01-29 14:05:37,026] {processor.py:163} INFO - Started process (PID=3046) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:05:37,029] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:05:37,030] {logging_mixin.py:109} INFO - [2022-01-29 14:05:37,030] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:05:38,222] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:05:38,240] {logging_mixin.py:109} INFO - [2022-01-29 14:05:38,239] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:05:38,272] {logging_mixin.py:109} INFO - [2022-01-29 14:05:38,271] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:05:38,291] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.271 seconds
[2022-01-29 14:06:08,403] {processor.py:163} INFO - Started process (PID=3077) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:06:08,405] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:06:08,406] {logging_mixin.py:109} INFO - [2022-01-29 14:06:08,406] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:06:09,222] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:06:09,480] {logging_mixin.py:109} INFO - [2022-01-29 14:06:09,480] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:06:09,497] {logging_mixin.py:109} INFO - [2022-01-29 14:06:09,497] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:06:09,512] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.114 seconds
[2022-01-29 14:06:39,645] {processor.py:163} INFO - Started process (PID=3107) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:06:39,652] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:06:39,654] {logging_mixin.py:109} INFO - [2022-01-29 14:06:39,653] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:06:41,099] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:06:41,109] {logging_mixin.py:109} INFO - [2022-01-29 14:06:41,109] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:06:41,129] {logging_mixin.py:109} INFO - [2022-01-29 14:06:41,129] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:06:41,140] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 1.506 seconds
[2022-01-29 14:07:11,254] {processor.py:163} INFO - Started process (PID=3147) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:07:11,256] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:07:11,257] {logging_mixin.py:109} INFO - [2022-01-29 14:07:11,257] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:07:12,070] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:07:12,080] {logging_mixin.py:109} INFO - [2022-01-29 14:07:12,079] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:07:12,099] {logging_mixin.py:109} INFO - [2022-01-29 14:07:12,099] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:07:12,109] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.859 seconds
[2022-01-29 14:07:42,669] {processor.py:163} INFO - Started process (PID=3180) to work on /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:07:42,672] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_gcs_dag_2.py for tasks to queue
[2022-01-29 14:07:42,673] {logging_mixin.py:109} INFO - [2022-01-29 14:07:42,673] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:07:43,459] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_gcs_dag_2']) retrieved from /opt/airflow/dags/data_ingestion_gcs_dag_2.py
[2022-01-29 14:07:43,469] {logging_mixin.py:109} INFO - [2022-01-29 14:07:43,468] {dag.py:2396} INFO - Sync 1 DAGs
[2022-01-29 14:07:43,487] {logging_mixin.py:109} INFO - [2022-01-29 14:07:43,487] {dag.py:2935} INFO - Setting next_dagrun for data_ingestion_gcs_dag_2 to 2022-01-02T06:00:00+00:00
[2022-01-29 14:07:43,497] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_gcs_dag_2.py took 0.832 seconds
